extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/runtime/src/main/java/io/quarkus/resteasy/reactive/jackson/CustomDeserialization.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.resteasy.reactive.jackson;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.lang.reflect.Type;
+import java.util.function.BiFunction;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+
+import io.smallrye.common.annotation.Experimental;
+
+/**
+ * Annotation that can be used on RESTEasy Reactive Resource method to allow users to configure Jackson deserialization
+ * for that method only, without affecting the global Jackson configuration.
+ */
+@Experimental(value = "Remains to be determined if this is the best possible API for users to configure per Resource Method Deserialization")
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.METHOD })
```

Comment:
We might want to enhance this in the future to also include adding the annotation on classes instead of just methods

Reply:
Yes, I thought doing it as part of this PR, but I think it's better to handle that as a follow-up pull request.